plugins/GSdx/GPU.cpp

```diff
@@ -107,17 +107,17 @@ EXPORT_C_(int32) GPUopen(void* hWnd)
 
 #endif
 
-	int renderer = theApp.GetConfig("Renderer", 1);
+	GS_RENDERER renderer = static_cast<GS_RENDERER>(theApp.GetConfig("Renderer", GS_DEFAULT_RENDERER));
 	int threads = theApp.GetConfig("extrathreads", 0);
 
 	switch(renderer)
 	{
 	default:
 	#ifdef _WINDOWS
-	case 0: s_gpu = new GPURendererSW(new GSDevice9(), threads); break;
-	case 1: s_gpu = new GPURendererSW(new GSDevice11(), threads); break;
+	case REND_D3D9_HW: s_gpu = new GPURendererSW(new GSDevice9(), threads); break;
```

Comment:
This might look strange but I was directly converting the value to an enum here. I don't know the purpose of that code. I also have the feeling that the function is at best not used, but for sure completely outdated. I could not find a reference to that function